### PR TITLE
docs: add installation doc (minikube support)

### PR DESCRIPTION
**Pull request by lianghao208**
add installation doc:
- support install kstone on minikube(Mac OS X and Linux amd64)

### CODE REVIEW

File: charts/README.md [hunk 1]
```diff
@@ -7,124 +7,18 @@
 - Prerequisites
   - Kubernetes version is between 1.14 and 1.20.
   - The version of Prometheus Operator is v0.49.0.
-- Apply for a cluster from [TKE](https://cloud.tencent.com/product/tke) or install [minikube](https://minikube.sigs.k8s.io/docs/start/).
+- Apply for a cluster from [TKE](https://cloud.tencent.com/product/tke) or install [Minikube](https://minikube.sigs.k8s.io/docs/start/).
```
Comment (tangcong) [inline charts/README.md:10]: It is recommended to also explain that kstone supports deployment in various cloud vendors and bare k8s cluster environments, and only the corresponding ingress rules need to be configured.

File: charts/README.md [hunk 2]
```diff
@@ -7,124 +7,18 @@
...
 - Requirements：
-  - Worker >= 4 vCPU 8 GB of Memory.
+  - For TKE: Worker >= 4 vCPU 8 GB of Memory.
```
Comment (tangcong) [inline charts/README.md:12]: pr #15  reduces the component request esource. Do we specify at least how many resources are recommended for the production environment and how much is required for the test environment?

File: docs/installation/minikube-amd64.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+- Prerequisites
+    - Kubernetes version is between 1.14 and 1.20.
+    - The version of Prometheus Operator is v0.49.0.
+- Requirements：
+    - Worker >= 2 vCPU 2 GB of Memory.
+    - Can access the managed etcd.
+
+## 2 Install Minikube
+
+#### Step 1：
```
Comment (tangcong) [inline docs/installation/minikube-amd64.md:16]: Is an optional step？

File: docs/installation/minikube-amd64.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+# Re-Login and Restart the Docker Server
+systemctl restart docker
+```
+
+#### Step 2：
+- [Install Minikube](https://minikube.sigs.k8s.io/docs/start/) (e.g. Linux amd64)
+    - Requirements：version <= 1.20.x
+```shell
+VERSION=v1.20.0
+curl -LO https://storage.googleapis.com/minikube/releases/${VERSION}/minikube-linux-amd64
```
Comment (tangcong) [inline docs/installation/minikube-amd64.md:39]: It’s better to give an address that Chinese users can download directly?

File: docs/installation/minikube-amd64.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+``` yaml
+// charts/charts/dashboard-api/values.yaml
+
+kube:
+  token: $token
+  target: kubernetes.default.svc.cluster.local:443
+```
+
+- Requirements：
+    - $token is the access credential TOKEN of the TKE cluster to be deployed.
```
Comment (tangcong) [inline docs/installation/minikube-amd64.md:86]: TKE cluster？It is best to give a method to get tokens in the minikube cluster.

**Review by tangcong — CHANGES_REQUESTED** [review]
good jobs. thanks. give some tips. PTAL.

File: docs/installation/minikube-amd64.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+passwd kstone
+
+su - kstone
+
+sudo groupadd docker
+
+sudo usermod -aG docker $USER
+
+# Re-Login and Restart the Docker Server
+systemctl restart docker
```
Comment (engow) [inline docs/installation/minikube-amd64.md:31]: I think the version of docker need to be specified.

File: docs/installation/minikube-amd64.md [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+git clone git@github.com:tkestack/kstone.git
+cd ./charts
+```
+
+#### Step 2：
+
+- Get Admin TOKEN from Minikube cluster
+
+```shell
+kubectget secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='default')].data.token}" -n kube-system|base64 --decode
```
Comment (tangcong) [inline docs/installation/minikube-amd64.md:78]: typo,kubectl get

File: docs/installation/minikube-amd64.md [hunk 8]
```diff
@@ -0,0 +1,192 @@
+# Kstone Installation
+
+[中文](README_CN.md)
```
Comment (tangcong) [inline docs/installation/minikube-amd64.md:3]: Link not found，it is better to provide a chinese installation document. If there is no time right now, we can remove this link first. thanks.

File: docs/installation/minikube-macos.md [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
+git clone git@github.com:tkestack/kstone.git
+cd ./charts
+```
+
+#### Step 2：
+
+- Get Admin TOKEN from Minikube cluster
+
+```shell
+kubectget secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='default')].data.token}" -n kube-system|base64 --decode
```
Comment (tangcong) [inline docs/installation/minikube-macos.md:72]: typo

**Review by tangcong — CHANGES_REQUESTED** [review]

**Comment by lianghao208** [comment]
@tangcong 
Done

File: docs/installation/minikube-amd64.md [hunk 10]
```diff
@@ -0,0 +1,191 @@
...
+              service:
+                name: kstone-grafana
+                port:
+                  number: 80
+EOF
+```
+
+- Expose Ingress Port
+```shell
+kubectl port-forward deployment/ingress-nginx-controller 8080:80 --namespace kube-system --address 0.0.0.0
```
Comment (tangcong) [inline docs/installation/minikube-amd64.md:149]: i got Error "from server (NotFound): deployments.apps "ingress-nginx-controller" not found".
namespace should be ingress-nginx. or is it related to your minikube version?

**Review by tangcong — CHANGES_REQUESTED** [review]

**Comment by lianghao208** [comment]
@tangcong 
You are right, It's ingress-nginx in v1.20.0. I've fixed this in doc.

**Review by tangcong — APPROVED** [review]
lgtm. thanks.

**Pull request merged**